### PR TITLE
Setting up API Endpoint for Analysis Results From Database

### PR DESCRIPTION
**Pull request by forkimenjeckayang**
Created an API endpoint route in app.py called "/api/fetch-data", when route is accessed , it executes a function called "fetch_data_from_database" found in the apiendpoint.py file. The results gotten from the database are displayed in a tabular form using the html template file called "table_template.html"

### CODE REVIEW

File: apiendpoint.py
Comment (stephane-segning) [inline apiendpoint.py:1]: This should come from the people implementing the database read and write.

**Review by stephane-segning — CHANGES_REQUESTED** [review]
Very good job Forkim! You're still missing some points

File: apiendpoint.py
Comment (forkimenjeckayang) [reply apiendpoint.py:1]: Thank you Sir. So i should remove the apiendpoint.py file , stage and commit the changes ?

Comment (stephane-segning) [reply apiendpoint.py:1]: No, you should cancel the changes done there. I mean, are you the one implementing that feature?

Comment (forkimenjeckayang) [reply apiendpoint.py:1]: Yes Sir, i believe so. The interface to display charts is quite different from the one to fetch analysis results from database which is the index.html file. When fetch results button is clicked it should access the API endpoint route get contents of the database and serve to the front end.

**Comment by forkimenjeckayang** [comment]
The "apiendpoint.py" file carries the logic to fetch analysis results from database when the "/api/fetch-data" route is accessed

**Comment by forkimenjeckayang** [comment]
Good morning Sir @stephane-segning , changes made on file as requested. If there's any other thing to do you can let me know

**Review by stephane-segning — APPROVED** [review]

**Comment by stephane-segning** [comment]
Good work @forkimenjeckayang

**Pull request merged**